GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -218,6 +219,11 @@ namespace gdjs {
 
         // Uncomment to display some debug metrics from Three.js.
         // console.log(threeRenderer.info);
+
+        const debugContainer = this._runtimeScene
+          .getDebuggerRenderer()
+          .getRendererObject();
+        if (debugContainer) pixiRenderer.render(debugContainer);
```

Comment:
You're doing a Pixi render, so you must ensure the state is ready for it, like done for layers.

This should be guarded like it's done for a 2D only rendering with 
```if (lastRenderWas3D) {
                    // Ensure the state is clean for PixiJS to render.
                    threeRenderer.resetState();
                    pixiRenderer.reset();
                  }
```

and set `lastRenderWas3D = false`
and done before the `if (!lastRenderWas3D)`